openhtf/io/frontend/src/app/utils.ts

```diff
@@ -41,6 +41,7 @@ export class SimplifyStatus extends MapPipe {
     'WAITING_FOR_TEST_START': 'WAITING',
     'STOP_WAIT': 'WAITING',
     'TIMEOUT': 'ERROR',
+    'ABORTED': 'ERROR',
```

Comment:
Hm? i was talking about the string mapping in your status renaming, you have keys in here that  don't exist anymore (won't actually break, but they should be removed), but you're also missing 'RUNNING' and 'COMPLETED', which will probably break, right?